src/wallet/wallet.h

```diff
@@ -699,6 +699,13 @@ class CWallet final : public WalletStorage, private interfaces::Chain::Notificat
      */
     int m_last_block_processed_height GUARDED_BY(cs_wallet) = -1;
 
+    /* Flag to indicate chain is still in initial block download. Set it to
+     * false to let wallet generating new seed without being connected to a node.
+     * Othersaid, a wallet evaluates chain being out of initial block download
+     * until it's proven false.
```

Comment:
Thanks for adding this documentation. Suggestion (if I understand correctly that _othersaid_ means _autrement dit_, which translates to _in other words_):
```diff
-    /* Flag to indicate chain is still in initial block download. Set it to
-     * false to let wallet generating new seed without being connected to a node.
-     * Othersaid, a wallet evaluates chain being out of initial block download
-     * until it's proven false.
+    /* Flag to indicate whether or not the chain is still in initial block
+     * download. Defaults to false to allow the wallet to generate new seeds
+     * without being connected to a node. In other words, the wallet assumes the
+     * chain is out of initial block download unless shown otherwise.
```